playback/jellyfin/src/main/kotlin/mediastream/UniversalAudioMediaStreamResolver.kt

```diff
@@ -0,0 +1,38 @@
+package org.jellyfin.playback.jellyfin.mediastream
+
+import org.jellyfin.playback.core.mediastream.MediaStream
+import org.jellyfin.playback.core.mediastream.MediaStreamResolver
+import org.jellyfin.playback.core.queue.item.QueueEntry
+import org.jellyfin.playback.jellyfin.queue.item.BaseItemDtoUserQueueEntry
+import org.jellyfin.sdk.api.client.ApiClient
+import org.jellyfin.sdk.api.client.extensions.universalAudioApi
+import org.jellyfin.sdk.model.api.BaseItemKind
+
+class UniversalAudioMediaStreamResolver(
+	val api: ApiClient,
+) : MediaStreamResolver {
+	override suspend fun getStream(queueEntry: QueueEntry): MediaStream? {
```

Comment:
Since this doesn't use anything from Android should it be in utils? A lot of our code for determining streams / codecs is currently there at the moment

Reply:
Everything related to the jellyfin API is in the playback/jellyfin module Media stream resolver implementations are one of those things. I don't see why that would be moved to the utils package in the app module.